liquid-rust-middle/src/global_env.rs

```diff
@@ -94,7 +94,18 @@ impl<'tcx> GlobalEnv<'tcx> {
             .iter()
             .map(|variant| self.default_variant_def(variant))
             .collect();
-        ty::AdtDef::transparent(adt_def.did, vec![], variants)
+        let generics = self.adt_def_generics(adt_def.did);
+        ty::AdtDef::transparent(adt_def.did, generics, vec![], variants)
+    }
+    pub fn adt_def_generics(&self, def_id: DefId) -> Vec<rustc_middle::ty::ParamTy> {
```

Comment:
There should be a space between these functions. I wonder why rustfmt is not complaining 🤔 

```suggestion
    }
    
    pub fn adt_def_generics(&self, def_id: DefId) -> Vec<rustc_middle::ty::ParamTy> {
```